wall_e/src/resources/cogs/reminders.py

```diff
@@ -49,22 +52,38 @@ def __init__(self, bot, config):
             logger.error("[Reminders __init__] enountered following exception when setting up PostgreSQL "
                          "connection\n{}".format(e))
 
-    @commands.command()
+    @commands.command(aliases=['remindmeon'])
     async def remindmein(self, ctx, *args):
         logger.info("[Reminders remindmein()] remindme command detected from user {}".format(ctx.message.author))
         parsed_time = ''
         message = ''
+        tz = _timezone
```

Comment:
what timezone is this supposed to represent exactly?

Reply:
The timezone the user is using; it's set to `_timezone` by default but changes if the user specifies something else
I could change it to something more specific like `user_timezone` or something though

Reply:
>  I could change it to something more specific like user_timezone or something though

please do

Reply:
Done